### PR TITLE
Added model resources for LayoutLM Issue#19848

### PR DESCRIPTION
**Pull request by avisinghal6**
Added resources to documentation of LayoutLM model as per Issue#19848.
@stevhliu

### CODE REVIEW

File: docs/source/en/model_doc/layoutlm.mdx [hunk 1]
```diff
@@ -73,12 +73,27 @@ image = Image.open(name_of_your_document).convert("RGB")
...
+LayoutLM for document-understanding using Keras & Hugging Face
+Transformers`](https://www.philschmid.de/fine-tuning-layoutlm-keras).
+
+- A notebook on how to [`fine-tune LayoutLM on the FUNSD dataset`](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/LayoutLM/Add_image_embeddings_to_LayoutLM.ipynb).
+
+- A notebook on how to [`fine-tune LayoutLMForSequenceClassification on the RVL-CDIP dataset`](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/LayoutLM/Fine_tuning_LayoutLMForSequenceClassification_on_RVL_CDIP.ipynb).
+
+- A blog post on how to [Fine-tuning LayoutLM for document-understanding using Hugging Face Transformers](https://www.philschmid.de/fine-tuning-layoutlm).
+
+- A notebook on how to [ fine-tune the LayoutLM model on the FUNSD dataset](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/Fine_tuning_LayoutLMForTokenClassification_on_FUNSD.ipynb).
```
Comment (stevhliu) [inline docs/source/en/model_doc/layoutlm.mdx:92]:
```suggestion
<PipelineTag pipeline="document-question-answering"/>

- A blog post on [fine-tuning
LayoutLM for document-understanding using Keras & Hugging Face
Transformers](https://www.philschmid.de/fine-tuning-layoutlm-keras).

- A blog post on how to [fine-tune LayoutLM for document-understanding using only Hugging Face Transformers](https://www.philschmid.de/fine-tuning-layoutlm).

- A notebook on how to [fine-tune LayoutLM on the FUNSD dataset with image embeddings](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/LayoutLM/Add_image_embeddings_to_LayoutLM.ipynb).

<PipelineTag pipeline="text-classification"/>

- A notebook on how to [fine-tune LayoutLM for sequence classification on the RVL-CDIP dataset](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/LayoutLM/Fine_tuning_LayoutLMForSequenceClassification_on_RVL_CDIP.ipynb).

<PipelineTag pipeline="token-classification"/>

- A notebook on how to [ fine-tune LayoutLM for token classification on the FUNSD dataset](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLM/Fine_tuning_LayoutLMForTokenClassification_on_FUNSD.ipynb).
```

**Review by stevhliu — COMMENTED** [review]
Thanks so much for your contribution, left a few comments on formatting!

File: docs/source/en/model_doc/layoutlm.mdx (hunk 1 above)
Comment (avisinghal6) [reply docs/source/en/model_doc/layoutlm.mdx:92]: I added the suggested changes

**Review by avisinghal6 — COMMENTED** [review]
Added the suggested changes

**Comment by stevhliu** [comment]
It looks like you might've only updated your branch with the latest changes on the `main` Hugging Face repo, I don't see any of the changes! 🙈

**Comment by avisinghal6** [comment]
Hi @stevhliu, you were right, i only updated my branch (sorry my bad ), i  think i have committed the changes now. Sorry for this.

File: docs/source/en/model_doc/layoutlm.mdx [hunk 2]
```diff
@@ -73,12 +73,33 @@ image = Image.open(name_of_your_document).convert("RGB")
...
 
-- For a demo which shows how to fine-tune [`LayoutLMForTokenClassification`] on the [FUNSD dataset](https://guillaumejaume.github.io/FUNSD/) (a collection of annotated forms), see [this notebook](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLM/Fine_tuning_LayoutLMForTokenClassification_on_FUNSD.ipynb).
-  It includes an inference part, which shows how to use Google's Tesseract on a new document.
+## Resources
 
-This model was contributed by [liminghao1630](https://huggingface.co/liminghao1630). The original code can be found
-[here](https://github.com/microsoft/unilm/tree/master/layoutlm).
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with LayoutLM. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
 
+<PipelineTag pipeline="token-classification" />
```
Comment (stevhliu) [inline docs/source/en/model_doc/layoutlm.mdx:80]: This tag can be removed since this section is about document understanding

**Review by stevhliu — COMMENTED** [review]
No worries at all @avisinghal6! Remember to revert the italics formatting from earlier (see [here](https://github.com/huggingface/transformers/pull/21377/files#r1092228692)). These should be the last changes and then we should be good to merge! 😄

**Comment by stevhliu** [comment]
Awesome, thank you so much for your contribution! 🤗  Everything looks good to me, pinging @sgugger for a final look!

**Review by stevhliu — APPROVED** [review]

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**